docs/api-guide.md

```diff
@@ -583,6 +583,47 @@ Put put =
         .build();
 ```
 
+##### Put with a condition
+
```

Comment:
I think it might be helpful to add something like this here.

You can write arbitrary conditions (e.g., a bank account balance must be equal to or more than zero) that you require a transaction to meet before being committed by having some checking logic in the transaction. Alternatively, you can also write simple conditions in a mutation operation, such as Put and Delete.

(If this is added, I think we need to update the next sentence a bit.)

Reply:
Thank you for the suggestion,  
To avoid redundancy in the explanation, I will merge the `Put with a condition` and `Delete with a condition` paragraphs.

Reply:
Fixed in https://github.com/scalar-labs/scalardb/pull/899/commits/46441d23a7772be58cd2ad2bfb887b71767f5b31